.github/workflows/ci-suite.yml

```diff
@@ -184,7 +184,7 @@ jobs:
         doxyfile-path: 'docs/Doxyfile'
 
     - name: gh-pages push
-      if: github.event_name == 'push' && github.event.ref == 'refs/heads/dev'
+      if: github.event_name == 'push' && github.event.ref == 'refs/heads/dev' && env.TGS_RELEASE_NOTES_TOKEN != ''
```

Comment:
This variable still needs to be set at the job or step level with
```yml
env:
  TGS_RELEASE_NOTES_TOKEN: ${{ secrets.TGS_RELEASE_NOTES_TOKEN }}
```

Reply:
np